DataFormats/L1TrackTrigger/interface/L1Track.h

```diff
@@ -0,0 +1,6 @@
+#include "DataFormats/L1TrackTrigger/interface/TTTrack.h"
```

Comment:
Was the include guard omitted on purpose? Other headers in this space seem to include it and I can't think of a good reason to not have it.

Reply:
yes that was an accident, thanks